apix-impl/src/test/java/eu/xenit/apix/tests/search/SearchServiceUnitTest.java

```diff
@@ -121,6 +122,27 @@ public void testOrderBy_withMultivalueProperty_throwsIllegalArgumentException()
         apixSearchServiceMocked.query(query);
     }
 
+    @Test
+    public void testLimitDefault() {
```

Comment:
Same rename here

Reply:
The parameter is actually called `limit` so I think the name should stay.